docs/extending/connectors.md

```diff
@@ -16,15 +16,28 @@ Connectors are a class which extends the base opsdroid Connector. The class has
 #### user_typing
 *user_typing* triggers the event message *user is typing* if the connector allows it. This method uses a parameter `trigger` that takes in a boolean value to trigger the event on/off.
 
-#### respond
-*respond* will take a Message object and return the contents to the chat service.
-
-#### react
-*react* will take a Message object and an emoji name and add this emoji as a Message reaction (if the chat service supports this feature).
-
 #### disconnect
 *disconnect* there is also an optional disconnect method that will be called upon shutdown of opsdroid. This can be used to perform any disconnect operations for the connector.
 
+
+### Handling Events
+
+Opsdroid supports different types of events, which can both be sent and received via connectors, for more information on the different types of events see the [events documentation](./events.md).
```

Comment:
I'm not sure if the link will work in readthedocs - because linking is a bit weird. I believe if you use `/events.md` it will work fine